otlp/src/otlp/logs/component.cpp

```diff
@@ -21,6 +22,18 @@ USERVER_NAMESPACE_BEGIN
 
 namespace otlp {
 
+namespace {
+SinkType GetSinkTypeFromString(const std::string& destination) {
+  if (destination == "both") {
+    return SinkType::kBoth;
+  }
+  if (destination == "default") {
+    return SinkType::kDefault;
+  }
+  return SinkType::kOtlp;
```

Comment:
you need an explicit check, otherwise a typo would silently default to otlp

Reply:
does enum in schema not mean implicit check on yaml config level?

Reply:
explicit is better than implicit here